cmd/root.go

```diff
@@ -32,18 +33,40 @@ var RootCmd = &cobra.Command{
 	Short: "A FTP server that persists all data to Beyond Storage.",
 	Long:  "A FTP server that persists all data to Beyond Storage.",
 	Run: func(cmd *cobra.Command, args []string) {
-		if versionFlag {
-			fmt.Fprintf(os.Stdout, "BeyondFTP version %s\n", constants.Version)
-			return
+		if cfgDebugFlag {
+			pprof.StartPP()
 		}
-
 		c := config.LoadConfigFromFilepath(cfgFileFlag)
 		s, err := server.NewFTPServer(c)
 		check.ErrorForExit("server init error", err)
 		StartServer(s)
 	},
 }
 
+var versionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Print Beyond-FTP version",
+	Long:  "Print Beyond-FTP version",
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Printf("BeyondFTP version %s", constants.Version)
+	},
+}
+
+// Execute adds all child commands to the root command sets flags appropriately.
+// This is called by main.main(). It only needs to happen once to the rootCmd.
+func Execute() {
+	if err := RootCmd.Execute(); err != nil {
+		check.ErrorForExit(constants.Name, err)
+	}
```

Comment:
Why remove the `os.Exit(-1)` here?

Reply:
`check.ErrorForExit` will exit if err not nil